src/Yahoo/Provider.php

```diff
@@ -23,6 +23,9 @@ class Provider extends AbstractProvider
      */
     protected function getAuthUrl($state)
     {
+        $this->scopes(['openid2']);
+        $parse_url = parse_url($this->redirectUrl);
```

Comment:
But you don't need all part here?

Reply:
Please limit the components you retrieve: ` parse_url($this->redirectUrl, PHP_URL_SCHEME | PHP_URL_HOST);`

Reply:
Also `$parse_url`:  variables are not noted in snake_case in this project, but in camelCase